### ISSUE TITLE
Multiple dex files define Lcom/google/android/gms/internal/zzne

### ISSUE THREAD

**Issue by rreusser** [teamrota/react-native-gmaps#10]
Hmm… I'm not sure where to go from here but encountered the following error on build. I'm also using [rn-geolocation](https://www.npmjs.com/package/rn-geolocation) which may or may not be contributing to an API version mismatch/conflict.

This seems vaguely related but didn't really solve the issue: http://stackoverflow.com/questions/29986206/unable-to-execute-dex-multiple-dex-files-define-lcom-google-android-gms-analyti

```
...
:app:dexDebug
Unknown source file : UNEXPECTED TOP-LEVEL EXCEPTION:
Unknown source file : com.android.dex.DexException: Multiple dex files define Lcom/google/android/gms/internal/zzne;
Unknown source file : 	at com.android.dx.merge.DexMerger.readSortableTypes(DexMerger.java:596)
Unknown source file : 	at com.android.dx.merge.DexMerger.getSortedTypes(DexMerger.java:554)
Unknown source file : 	at com.android.dx.merge.DexMerger.mergeClassDefs(DexMerger.java:535)
Unknown source file : 	at com.android.dx.merge.DexMerger.mergeDexes(DexMerger.java:171)
Unknown source file : 	at com.android.dx.merge.DexMerger.merge(DexMerger.java:189)
Unknown source file : 	at com.android.dx.command.dexer.Main.mergeLibraryDexBuffers(Main.java:502)
Unknown source file : 	at com.android.dx.command.dexer.Main.runMonoDex(Main.java:334)
Unknown source file : 	at com.android.dx.command.dexer.Main.run(Main.java:277)
Unknown source file : 	at com.android.dx.command.dexer.Main.main(Main.java:245)
Unknown source file : 	at com.android.dx.command.Main.main(Main.java:106)

:app:dexDebug FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:dexDebug'.
> com.android.ide.common.process.ProcessException: org.gradle.process.internal.ExecException: Process 'command '/Library/Java/JavaVirtualMachines/jdk1.8.0_66.jdk/Contents/Home/bin/java'' finished with non-zero exit value 2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

**Comment by rreusser** [comment]
Hmm… I initially thought it was a totally different error, but this may be related to #6. `react-native-gmaps/android/app/build.gradle` has:

```
compile 'com.google.android.gms:play-services:8.1.0'
```

which is the one that pulls in everything, right?

**Comment by rreusser** [comment]
Or possibly a conflict with [`rn-geolocation/android/build.gradle`](https://github.com/Tiagojdferreira/rn-geolocation/blob/master/android/build.gradle#L25):

```
dependencies {
    compile 'com.facebook.react:react-native:0.12.+'
    compile 'com.google.android.gms:play-services-location:8.3.+'
}
```

**Comment by rreusser** [comment]
Doesn't quite work yet but the compilation is solved by #9, so closing this.
